### ISSUE TITLE
CompilerException java.lang.RuntimeException: No such var: enc/now-nano*

### ISSUE THREAD

**Issue by ncalexan** [ptaoussanis/tufte#8]
I'm sure I'm doing something wrong, but I don't know what.  My lein configuration in `project.clj` includes:

```cloure
...
:dependencies [[org.clojure/clojurescript "1.9.89"]
               [org.clojure/clojure "1.8.0"]
               [org.clojure/core.async "0.2.385"]
               [com.taoensso/timbre "4.7.0"]
               [com.taoensso/tufte "1.0.2"]
               ....
```
However, when I try to use tufte, I get an error as follows:
```clojure
user=> (require '[taoensso.tufte :as tufte :refer [defnp p profiled profile]])

CompilerException java.lang.RuntimeException: No such var: enc/now-nano*, compiling:(taoensso/tufte/impl.clj:154:20)
```
Am I missing something?  Is the transitive encore dependency stale?

**Comment by ncalexan** [comment]
I get the same exception with 1.0.1 but I can require 1.0.0.  Unfortunately, I still get `No such var: taoensso.tufte.impl` when using `p`.

**Comment by ncalexan** [comment]
OK -- 1.0.2 works if I remove the `timbre` reference.  Perhaps an incompatibility with the underlying versions of encore?  This presents like #3, but here I'm quite sure I'm not making an error in the `:require` statement: I'm not referring to `timbre` anywhere in the code at all!

**Comment by ptaoussanis** [comment]
Hi there,

The `No such var: enc/now-nano*` error should be fixed from Tufte 1.0.1 so this looks like a dependency conflict and/or stale build artifacts.

General strategy:
1. Make sure you've got all the latest dependencies in your project.clj (I recommend https://github.com/xsc/lein-ancient)
2. Run `lein clean` to make sure you've cleared out old build artifacts
3. Then run `lein deps :tree` to see if you've got any dependency conflicts.

Cheers!

**Comment by ptaoussanis** [comment]
Oh, in case it turns out you do have a dep conflict see here: https://github.com/ptaoussanis/encore/blob/master/DEP-CONFLICT.md
